### PR TITLE
Feat/jsonc

### PR DESCRIPTION
**Pull request by smoliji**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Configuru is a library for configuration management. Merge default project confi
  - :blue_heart: Typescript friendly
  - :muscle: Designed for multi-environment apps
  - :see_no_evil: Anonymized configuration for logger
+ - :eye_in_speech_bubble: JSONC support
```
Comment (grissius) [inline README.md]: Alas, this does not seem to work 
image

File: src/test/sandbox/default.jsonc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    // Inline comment
+    "foo": "default",
+    /*
+        Block comment
+    */
+    "bar": 1,
+    "baz": 2,
+    "defaultConfig": true
+}
```
Comment (grissius) [inline src/test/sandbox/default.jsonc]: Missing newline

File: README.md (hunk 1 above)
Comment (smoliji) [reply README.md]: ok GH supports eye_speech_bubble, GL supports eye_in_speech_bubble  🤯.

Can I use unicode for this one? :pray:

File: src/test/sandbox/default.jsonc (hunk 2 above)
Comment (smoliji) [reply src/test/sandbox/default.jsonc]: Fixed. Was a 1:1 copy of an already present `default.json`

File: README.md (hunk 1 above)
Comment (grissius) [reply README.md]: Sure :+1:

File: src/test/sandbox/default.jsonc (hunk 2 above)
Comment (grissius) [reply src/test/sandbox/default.jsonc]: Whoopsie, my bad.

Comment (grissius) [reply src/test/sandbox/default.jsonc]: Thanks.

File: README.md (hunk 1 above)
Comment (smoliji) [reply README.md]: Fixed :checkered_flag:

**Pull request merged**